src/components/Book.js

```diff
@@ -6,17 +6,56 @@ import { removeBook } from '../Redux/books/books';
 const Book = ({ title, author, id }) => {
   const dispatch = useDispatch();
 
-  const handleRemoveBook = (id) => {
+  const handleRemoveBook = () => {
     dispatch(removeBook(id));
   };
 
   return (
-    <div className="book">
-      <h1>{title}</h1>
-      <p>{author}</p>
-      <button type="button" onClick={() => handleRemoveBook(id)}>
-        Remove
-      </button>
+    <div className="book-container">
+      <div className="book">
+        <div className="book-cover">
+          <h4 className="book-category">Fiction</h4>
+          <h2 className="book-title">{title}</h2>
+          <p className="book-author">{author}</p>
+        </div>
+        <div className="action-buttons">
+          <button type="button" className=" btn-primary">
+            Comments
+          </button>
+          <div className="divider" />
+          <button
+            type="button"
+            className="btn-primary"
+            onClick={() => handleRemoveBook()}
+          >
+            Remove
+          </button>
+          <div className="divider" />
+          <button type="button" className="btn-primary">
+            Edit
+          </button>
+        </div>
+      </div>
+
+      <div className="progress-container">
+        <div className="progress">
+          <div className="progress-circle-container">
+            <div className="circular-progress" />
+          </div>
+          <div className="progress-status">
+            <p className="progress-value">60%</p>
+            <p className="completed">Completed</p>
+          </div>
+        </div>
+        <div className="progress-divider" />
+        <div className="current-chapter">
+          <p className="current-chapter-title">CURRENT CHAPTER</p>
+          <p className="current-chapter-number">Chapter 17</p>
+          <button type="button" className="btn-update">
+            UPDATE PROGRESS
+          </button>
+        </div>
+      </div>
```

Comment:
- [ ] @ReemMohamedAbdelfatah kindly check on your styling the font  differs from the given design, also the color of the navbar items home and categories differs from the design given, check the comparison below for better understanding:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/216696287-aea352fd-e1c2-4086-bcc2-889cb8290553.png)  | ![image](https://user-images.githubusercontent.com/61408860/216696512-69377955-5888-4007-ba4f-d48524ccfd46.png)

Reply:
@danmainah Done👌